### DISCUSSION TITLE
Hello, everyone. I want to set the "Export Data" button as unselectable when right-clicking on the table. What is the module or java file corresponding to this function?

### DISCUSSION THREAD

**Discussion by hope-courage** [dbeaver/dbeaver#18710] [DBeaver Development]
Hello, everyone. I want to set the "Export Data" button as unselectable when right-clicking on the table. What is the module or java file corresponding to this function?

Select Table  -》 Right click  -》 export data

**Comment by ShadelessFox** [comment]
Hello @hope-courage,

The action's enablement is controlled by the handler's `enabledWhen` condition:

https://github.com/dbeaver/dbeaver/blob/9f128ed65762c384e182f8891f1e9e3be12c5761/plugins/org.jkiss.dbeaver.data.transfer.ui/plugin.xml#L36-L43
